docs/learn/advanced/03-node.md

```diff
@@ -93,4 +93,4 @@ Upon starting, the node will bootstrap its RPC and P2P server and start dialing
 
 ## Other commands
 <!-- markdown-link-check-disable-next-line -->
-To discover how to concretely run a node and interact with it, please refer to our [Running a Node, API and CLI](../../user/run-node/01-run-node.md#configuring-the-node-using-apptoml-and-configtoml) guide.
+To discover how to concretely run a node and interact with it, please refer to our [Running a Node, API and CLI](https://github.com/cosmos/cosmos-sdk-docs/blob/main/docs/user/run-node/01-run-node.md#configuring-the-node-using-apptoml-and-configtoml) guide.
```

Comment:
we should leave the direct linking since this is used in the docs page. This will send users to the repo which is not desired

Reply:
sure I will revert this. However, a lot of file paths are not there anymore (ex. docs/user/run-node/01-run-node.md) so we still need other ways to fix this